operator/config/watcher/kustomization.yaml

```diff
@@ -11,6 +11,21 @@ patches:
   - patch: |-
       - op: add
         path: /spec/template/spec/containers/0/args/-
-        value: --enable-kcp-watcher
+        value: --enable-kcp-watcher      
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --skr-watcher-path=/skr-webhook      
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --requeue-success-interval=5m      
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --requeue-waiting-interval=5m      
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --requeue-failure-interval=5m
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --kcp-addr=34.76.239.193:80
```

Comment:
do we have an issue cover this address problem? to reach it from fixed DNS name instead of IP?

Reply:
Not yet good point. That is a whole another feature implementation. Should not be a part of this PR.